frontend/apps/web/src/model/CI.ts

```diff
@@ -0,0 +1,42 @@
+// generated from JSON using https://app.quicktype.io/
```

Comment:
Can we do something about the `ci.json` file? There are 158k lines here, I'm not sure how will this impact the performance.

Reply:
hm, e2e-tests time is the same as before, will check it out, maybe it's fine 😄 .